lib/phobos.rb

```diff
@@ -55,38 +55,51 @@ def create_exponential_backoff(backoff_config = nil)
       ExponentialBackoff.new(min, max).tap { |backoff| backoff.randomize_factor = rand }
     end
 
+    # :nodoc:
     def configure_logger
-      log_file = config.logger.file
       ruby_kafka = config.logger.ruby_kafka
-      date_pattern = '%Y-%m-%dT%H:%M:%S:%L%zZ'
-      json_layout = Logging.layouts.json(date_pattern: date_pattern)
-
-      stdout_layout = if config.logger.stdout_json == true
-        json_layout
-      else
-        Logging.layouts.pattern(date_pattern: date_pattern)
-      end
-
-      appenders = [Logging.appenders.stdout(layout: stdout_layout)]
 
       Logging.backtrace(true)
       Logging.logger.root.level = silence_log ? :fatal : config.logger.level
-
-      if log_file
-        FileUtils.mkdir_p(File.dirname(log_file))
-        appenders << Logging.appenders.file(log_file, layout: json_layout)
-      end
+      appenders = logger_appenders
```

Comment:
I think you can remove this, and change line 73 to `@ruby_kafka_logger.appenders = logger_appenders`

Reply:
It gets used twice, once on line 73 and once on line 82, so it's memoized this way :)